kibicara/platforms/twitter/model.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2020 by Cathy Hu <cathy.hu@fau.de>
+#
+# SPDX-License-Identifier: 0BSD
+
+from kibicara.model import Hood, Mapping
+from ormantic import Boolean, Integer, ForeignKey, Model, Text
+
+
+class Twitter(Model):
+    id: Integer(primary_key=True) = None
+    hood: ForeignKey(Hood)
+    dms_since_id: Integer(allow_null=True) = None
+    mentions_since_id: Integer(allow_null=True) = None
+    access_token: Text()
+    access_token_secret: Text()
+    successful_verified: Boolean() = False
+
+    class Mapping(Mapping):
+        table_name = 'twitterbots'
```

Comment:
Why does the table name differ so much from the class name? Not a strong objection though, doesn't make much of a difference.